site/plugin.md

```diff
@@ -56,6 +56,30 @@ then run your containers using the Docker command-line:
     64 bytes from 10.32.0.2: icmp_seq=2 ttl=64 time=0.052 ms
 
 
+###<a name="multi"></a>Creating multiple Docker Networks
+
+Docker lets you create multiple independent networks and attach
+different sets of containers to each network. However, to coordinate
```

Comment:
--minor tweaks--
Docker enables you to create multiple independent networks and attach different sets of containers to each network.  However, coordinating this between hosts, Docker requires that configure a "key-value store"  <-- missing a quote on the end and also tweaked that sentence.